### PR TITLE
chore: all artifactory endpoints on master are switched to lf

### PR DESCRIPTION
**Pull request by nstng**
## Summary

This PR changes all artifactory endpoints on master to LF. See #14495 for the overall migration issue to the LF artifactory.

See different commits for a clustered overview what changed.

### Approach

* search for `artifactory.magmacore`
  * exclude `version-1.5.X/**,version-1.6.X/**,version-1.7.0/**,version-1.8.0/**,upgrade_1_5.md`
* for docker the URL pattern changes
  * `docker.artifactory.magmacore.org/foo` -> `linuxfoundation.jfrog.io/magma-docker/foo`
  * `agw-test.artifactory.magmacore.org` -> `linuxfoundation.jfrog.io/agw-test`
  * ...
* there is currently no virtual `helm` repo on LF (points to `helm-prod` on MC), so for now:
  * `https://artifactory.magmacore.org/artifactory/helm/` -> `https://linuxfoundation.jfrog.io/artifactory/magma-helm-prod/`

### not changed

* MC artifactory in double push workflows
* MC artifactory in workflows where double push is yet not implemented

## Test Plan

CI

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by nstng** [comment]
Open for review so that the large amount of reviewer-groups can already have a look - but still marked as DONOTLAND because this depends on double push workflows to the LF artifactory.

**Review by MoritzThomasHuebner — APPROVED** [review]

**Review by wolfseb — APPROVED** [review]

File: docs/readmes/orc8r/dev_minikube.md [hunk 1]
```diff
@@ -139,8 +139,8 @@ A minimal values file is at `${MAGMA_ROOT}/orc8r/cloud/helm/orc8r/examples/minik
 
 This section describes how to install based on local charts. However, you can also install charts from the official chart repositories
 
-- Stable: <https://artifactory.magmacore.org/artifactory/helm/>
-- Test: <https://artifactory.magmacore.org/artifactory/helm-test/>
+- Stable: <https://linuxfoundation.jfrog.io/artifactory/helm-prod/>
+- Test: <https://linuxfoundation.jfrog.io/artifactory/helm-test/>
```
Comment (sebathomas) [inline docs/readmes/orc8r/dev_minikube.md]: Both of these redirect me to a login page. Is it really helm-x or magma-helm-x?

Comment (nstng) [reply docs/readmes/orc8r/dev_minikube.md]: missed adapting in the docs commit - thanks for finding this

Comment (nstng) [reply docs/readmes/orc8r/dev_minikube.md]: changed

**Review by sebathomas — APPROVED** [review]

**Review by tmdzk — APPROVED** [review]
This looks good to me excellent job @nstng 
I checked to download: 
- Docker (through virtual and all 4)
- apt
- blob 
and everything was working fine 

I think at some point we should just merge it and see what's breaking 

cc @sebathomas @maxhbr

File: .devcontainer/Dockerfile [hunk 2]
```diff
@@ -72,7 +72,7 @@ RUN echo "Install general purpose packages" && \
 WORKDIR /usr/local
 ARG GOLANG_VERSION="1.19.3"
 RUN GO_TARBALL="go${GOLANG_VERSION}.linux-amd64.tar.gz" \
-    && curl https://artifactory.magmacore.org/artifactory/generic/${GO_TARBALL} --remote-name --location \
+    && curl -L https://linuxfoundation.jfrog.io/artifactory/magma-blob/${GO_TARBALL} --remote-name --location \
```
Comment (voisey) [inline .devcontainer/Dockerfile]: Why was this flag introduced here and elsewhere? Isn't it the same as `--location`?

Comment (sebathomas) [reply .devcontainer/Dockerfile]: We tried downloading from go.dev directly at some intermediate step.  But you're right, we already have a `--location` flag so no reason to add `-L`.

Comment (sebathomas) [reply .devcontainer/Dockerfile]: Fixed.

Comment (voisey) [reply .devcontainer/Dockerfile]: Thanks!

**Review by voisey — APPROVED** [review]

**Review by xbend — APPROVED** [review]
dp LGTM

File: .github/workflows/lte-integ-test-containerized.yml [hunk 3]
```diff
@@ -81,7 +81,7 @@ jobs:
         run: cat lte/gateway/docker/docker-compose.yaml
       - name: Run the integration test
         env:
-          DOCKER_REGISTRY: docker-ci.artifactory.magmacore.org/
+          DOCKER_REGISTRY: linuxfoundation.jfrog.io/magma-docker-test/
```
Comment (jheidbrink) [inline .github/workflows/lte-integ-test-containerized.yml]: FYI: This has a conflict with #14534 where I used `inputs.registry` and inlined it into the `fab` command below.

File: experimental/cloudstrapper/playbooks/roles/ghz-platform/vars/main.yaml [hunk 4]
```diff
@@ -36,7 +36,7 @@ load_tests:
         id: 16
 
 tag: latest
-docker_repo: agw-test.artifactory.magmacore.org
+docker_repo: linuxfoundation.jfrog.io/agw-test
```
Comment (jheidbrink) [inline experimental/cloudstrapper/playbooks/roles/ghz-platform/vars/main.yaml]:
```suggestion
docker_repo: linuxfoundation.jfrog.io/magma-docker-agw-test
```

File: lte/gateway/docker/ghz/Dockerfile [hunk 5]
```diff
@@ -1,5 +1,5 @@
 # AGW test docker private repository location
-ARG REPO_LOCATION=agw-test.artifactory.magmacore.org
+ARG REPO_LOCATION=linuxfoundation.jfrog.io/agw-test
```
Comment (jheidbrink) [inline lte/gateway/docker/ghz/Dockerfile]:
```suggestion
ARG REPO_LOCATION=linuxfoundation.jfrog.io/magma-docker-agw-test
```

**Review by jheidbrink — APPROVED** [review]

File: .github/workflows/lte-integ-test-containerized.yml (hunk 3 above)
Comment (sebathomas) [reply .github/workflows/lte-integ-test-containerized.yml]: I will rebase after that one is merged.

**Comment by sebathomas** [comment]
@jheidbrink The DCO check is red, you need to sign your commit.

File: orc8r/gateway/docker/Dockerfile [hunk 6]
```diff
@@ -37,11 +37,10 @@ FROM ubuntu:focal AS gateway_python
 
 # Add the magma apt repo
 RUN apt-get update && \
-    apt-get install -y apt-utils software-properties-common apt-transport-https
-COPY orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub /tmp/jfrog.pub
+    apt-get install -y apt-utils software-properties-common apt-transport-https wget
 COPY cwf/gateway/deploy/roles/ovs/files/magma-preferences /etc/apt/preferences.d/
-RUN apt-key add /tmp/jfrog.pub && \
-    apt-add-repository "deb https://artifactory.magmacore.org/artifactory/debian-test focal-ci main"
+RUN RUN wget -qO - https://linuxfoundation.jfrog.io/artifactory/api/security/keypair/magmaci/public | apt-key add - && \
```
Comment (sebathomas) [inline orc8r/gateway/docker/Dockerfile]: Duplicate `RUN` (thanks @voisey)

Comment (voisey) [inline orc8r/gateway/docker/Dockerfile:42]: There are duplicate "RUN"s here, which I'm guessing is not intentional

**Comment by sebathomas** [comment]
@jheidbrink Can you amend your commit to fix the DCO check? Then we can rebase this PR. :)

**Pull request merged**